### ISSUE TITLE
Unable to work with config source properties

### ISSUE THREAD

**Issue by jakkaj** [CodeConnect/AliveFeedback#81]
When using configSource to split out config in to separate files Alive shows an error and does not run. This code runs in the standard VS test runner. 

``` XML
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <appSettings configSource="Config_Boost_Dev.config">

  </appSettings>
```

```
[Error] Unhandled AggregateException: One or more errors occurred.
An exception was thrown while invoking the constructor 'Void .ctor()' on type 'WebConfig'. ---> Unable to open configSource file 'Config_XXX_Dev.config'. (xxx.config line 3) (See inner exception for details.)
Unable to open configSource file 'Config_XXXt_Dev.config'. (XXX.Tests.config line 3) 
```

**Comment by AmadeusW** [comment]
Hi @jakkaj, I'm sorry about this. 
I will fix this issue this week.

**Comment by AmadeusW** [comment]
I see that `configSource` is set to `"Config_Boost_Dev.config"` but reported error mentions `'Config_XXX_Dev.config'. (xxx.config line 3)`

Is this the actual error message, or does the error say `Config_Boost_Dev.config`?

**Comment by jakkaj** [comment]
Hi - yes sorry, I did modify the string for the post. It was originally
Config_boost_dev.config.

**Comment by AmadeusW** [comment]
I've fixed this for a limited case (hopefully yours) where you're using `configSource` or `file` property to load configuration from another file **in the same directory as the .csproj file**. Except an update later today!

We are debating what to do with cases where the `.config` file is located somewhere else. We're reluctant to parse configuration files looking for `configSource` or `file` properties, because this would considerably slow down Alive

**Comment by AmadeusW** [comment]
We've just released the [1.3.2 Update](https://github.com/CodeConnect/AliveFeedback/wiki/Changelog-1.3.2) which should resolve your problem. Let me know if you have any issues!
